server/database/models/manufacturers.js

```diff
@@ -0,0 +1,50 @@
+export function getAttributes(sequelize, DataTypes) {
+  return {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    name: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    origin_country: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    createdAt: {
+      field: 'created_at',
+      type: DataTypes.DATE,
+      allowNull: true,
+      defaultValue: sequelize.fn('now')
+    },
+    updatedAt: {
+      field: 'updated_at',
+      type: DataTypes.DATE,
+      allowNull: true
+    },
+    deletedAt: {
+      field: 'deleted_at',
+      type: DataTypes.DATE,
+      allowNull: true
+    }
+  };
+}
+
+export function model(sequelize, DataTypes) {
+  const manufacturers = sequelize.define('manufacturers', getAttributes(sequelize, DataTypes), {
+    tableName: 'manufacturers',
+    paranoid: true,
+    timestamps: true
+  });
+
+  manufacturers.associate = function(models) {
+    manufacturers.products = manufacturers.hasOne(models.products, {
```

Comment:
Each manufacturer can have multiple products, hence we will need a manufacturers.hasMany products association, not hasOne